README.md

```diff
@@ -138,6 +138,9 @@ Health checks will be repeated until success, and the interval can be configured
 
 It is currently possible to have expressions like `"test \"$(systemctl list-units --failed --no-legend --no-pager |wc -l)\" -eq 0"` (count number of failed systemd units, fail if non-zero) as the first argument in a cmd-healthcheck. This works, but is discouraged, and might break at any time.
 
+## Disabling morph
+
+In case you need to prevent morph to deploy onto system temporarily, just touch the /.morph_disabled file.
```

Comment:
1.  I'm partial towards calling it `/.morph_ignore` instead. Thoughts?
2.  Maybe we can instead write something like "Morph will ignore a host if the file `/.morph_ignore` is present on it.`

Reply:
Sure, _cordon_ works for me. I have used morph_disabled as I remember puppet having such file in early version as /.puppet_disabled.